lambda/src/app/lambdas/list/lambdas.component.html

```diff
@@ -1,5 +1,5 @@
 <section class="fd-section">
-  <fd-action-bar>
+  <fd-action-bar class="y-fd-action-bar">
```

Comment:
Since you removed the styles for `y-fd-action-bar`, we don't need these classes in html templates.

```suggestion
  <fd-action-bar>
```